src/EmbeddedVsix/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="SonarLint.AdditionalFiles.95521f6c-c3e6-460d-b0e6-aa64acd2fdb1" Version="1.0" Language="en-US" Publisher="SonarSource" />
+        <Identity Id="SonarLint.AdditionalFiles.95521f6c-c3e6-460d-b0e6-aa64acd2fdb1" Version="4.34.0.0" Language="en-US" Publisher="SonarSource" />
```

Comment:
The changes to the manifests are the result of building the `ChangeVersion` project.